crates/astria-sequencer-relayer/src/types.rs

```diff
@@ -167,6 +168,12 @@ pub struct SequencerBlockData {
     pub rollup_txs: HashMap<Namespace, Vec<IndexedTransaction>>,
 }
 
+impl Hash for SequencerBlockData {
```

Comment:
This violates the assumptions of the hash trait: https://doc.rust-lang.org/std/hash/trait.Hash.html#implementing-hash

This is also noted in the docs for priority queue:  https://docs.rs/priority-queue/latest/priority_queue/

Reply:
Updated.